decidim-core/app/cells/decidim/activity_cell.rb

```diff
@@ -90,6 +90,11 @@ def user
       model.user_lazy
     end
 
+    def author
+      return unless show_author?
+      cell "decidim/author", UserPresenter.new(user)
```

Comment:
Not all authors are users.

Reply:
`card_for present(user)` ?

Reply:
I tried this and it didn't work, as it tried to render the full-fledged card. Also, author in this context will always be a `Decidim::UserBaseEntity`, as I'm using the `user` field on `Activity`, which is not polymorphic.

Reply:
Ah, OK!